### PR TITLE
Add graphql service class

### PR DESCRIPTION
**Pull request by ttrig**

### CODE REVIEW

**Review by emil-nasso — APPROVED** [review]
:tada:

File: src/Graphql/Service.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    protected int $timeout = 10;
+
+    public function __construct()
+    {
+        $config = config(static::configKey());
+
+        $this->url = $config['url'];
+        $this->token = $config['token'];
+    }
```
Comment (olvert) [inline src/Graphql/Service.php:25]: Wondering if it should be possible to override the default config key as an argument in the constructor but maybe it's enough to add it if the need arises. Or maybe you should just use the `Butler\Graphql\Client` class in that case?

**Review by olvert — APPROVED** [review]

File: src/Graphql/Service.php (hunk 1 above)
Comment (ttrig) [reply src/Graphql/Service.php:25]: Instead of extending `static::configKey()`?

Comment (olvert) [reply src/Graphql/Service.php:25]: Ahh didn't think of that, forget what I said 🙃

**Pull request merged**